platform/security/cas/security-cas-tokenvalidator/src/main/java/ddf/security/cas/WebSSOTokenValidator.java

```diff
@@ -14,6 +14,7 @@
 package ddf.security.cas;
 
 import java.nio.charset.Charset;
+import java.util.Base64;
```

Comment:
However, we also need to be interoperable with other systems that are pushing binary security tokens to us and we need to be able to push our token to other STS implementations. If we are encoding it in such a way that it is not compliant with the spec, that would be bad. We need to compare the tokens created with each method and make sure they are identical before committing to the `j.u.Base64` version.